homeassistant/const.py

```diff
@@ -399,6 +399,9 @@ class Platform(StrEnum):
 _DEPRECATED_DEVICE_CLASS_PM25: Final = DeprecatedConstant(
     "pm25", "SensorDeviceClass.PM25", "2025.1"
 )
+_DEPRECATED_DEVICE_CLASS_PM4: Final = DeprecatedConstant(
+    "pm4", "SensorDeviceClass.PM4", "2025.1"
+)
```

Comment:
I don't think you should add a new deprecated constant. People who need this for the first time should use the new one.

Reply:
Removed